testing/integration/logs_ingestion_test.go

```diff
@@ -104,15 +104,11 @@ func testMonitoringLogsAreShipped(
 ) {
 	// Stage 1: Make sure metricbeat logs are populated
 	t.Log("Making sure metricbeat logs are populated")
-	require.Eventually(t,
```

Comment:
Will removing the eventually here make this flaky again?

Reply:
Ah I see there is a require eventually inside findESDocs.

Reply:
Exactly.